src/wwwroot/template-editor/adapterService.js

```diff
@@ -296,13 +296,44 @@
      */
     function campaignSaveChangesAsTemplate(params) {
       traceAdapterCall(campaignSaveChangesAsTemplate, arguments);
-      var savedCampaignAsTemplate = {
-        Success: true,
-        ErrorMessage: '',
-        UrlToRedirect: 'http://www.google.com/',
-        IdTemplate: 1
-      }
-      return savedCampaignAsTemplate;
+      return saveTemplateContentChanges(params).then(function() {
+        var url = $window.location.origin + '/#/templates';
+        var savedCampaignAsTemplate = {
+          Success: true,
+          ErrorMessage: '',
```

Comment:
Success and ErrorMessage make me suppose that these properties are used to deal with errors (in place of rejection), could you verify and use them to deal with expected and unexpected errors?